### ISSUE TITLE
Set Up Monitors for EP Merge App errors

### ISSUE THREAD

**Issue by amylai-va** [department-of-veterans-affairs/abd-vro#2493]
## **User Story**
As a member of the EE team, I want LHDI Datadog monitors set up for #2057 so that we can be alerted in Slack when these errors occur.

## **Acceptance Criteria**
- A monitor is set up for health check
- A monitor is set up for errors
- Monitors are set up for BIP and BGS
  - If any job completes in error, an alert should be sent
- In Dev, the threshold is set to: 1 error
- In Prod, the threshold is set to: x errors

**Considerations**
- How to minimize false positives?

## Notes about this work
- Additional background: https://github.com/department-of-veterans-affairs/abd-vro/issues/2240
- Related slack thread: https://dsva.slack.com/archives/C01BVF2A3V0/p1706728992131979

**Comment by amylai-va** [comment]
Notes from meeting with Tyler from LHDI:
* Useful resource: https://hub.dev.lighthouse.va.gov/docs/default/component/delivery-platform-docs (need to be on VA network)
* LHDI doesn't set up DD for us, but we can reach out to troubleshoot issues
* Summary of our request and his response:
  * We want to use DD to scan through logs that our app is producing and surface certain error messages to Slack
  * One of the other teams building on VRO is doing this. It's a little noisy. DD seems to be best at surfacing outages, producing two events: one for the outage and one when it's recovered
  * Is DD the best tool or is there another LHDI offering for what we want to do? In flight data would best be tracked somewhere else other than DD (didn't catch what it was)
  * **Recommendation**: set up a separate meeting to dive deeper into this topic

cc/ @dfitchett @nanotone @meganhicks

**Comment by dfitchett** [comment]
Blocked by 
- #2555 
- #2556

**Comment by dfitchett** [comment]
Currently waiting on the datadog DD_SITE and DD_API_KEY to be put in helm templates. 
* slack [thread](https://dsva.slack.com/archives/C01Q7979Z7D/p1707432385994699)
* #2593

**Comment by dfitchett** [comment]
Above has been resolved, but the behavior is unconfirmed because there is a new issue preventing deployment to VRO envs (see [thread](https://dsva.slack.com/archives/C01Q7979Z7D/p1707858065666779)).

**Comment by dfitchett** [comment]
Work with custom metrics are potentially blocked (see [thread](https://dsva.slack.com/archives/C01Q7979Z7D/p1708712624707489)), however, Employee Experience monitors have been configured to alert from data groomed from EP Merge Log output. 

Current monitors can be found [here](https://lighthousedi.ddog-gov.com/monitors/manage?q=team%3Aemployee-experience&order=desc). See tables below for direct links to monitors.

#### EP Merge
[DD Dashboard ](https://lighthousedi.ddog-gov.com/dashboard/unv-vap-yeb/employee-experience-ep-merge-vro-platform?fromUser=true&refresh_mode=sliding&tpl_var_namespace%5B0%5D=va-abd-rrd-sandbox&tpl_var_namespace%5B1%5D=va-abd-rrd-dev&tpl_var_namespace%5B2%5D=va-abd-rrd-qa&view=spans&from_ts=1708126487694&to_ts=1708731287694&live=true)

| name | env | purpose | alert channel |
| --- | --- | --- | --- | 
| [[Synthetics] PROD - Health Check - EP Merge App](https://lighthousedi.ddog-gov.com/monitors/194109) | prod | health check | [#benefits-ee-ep-merge-alerts](https://dsva.slack.com/archives/C067L5WK9K6)
| [PROD - EP Merge App - Jobs are Failing](https://lighthousedi.ddog-gov.com/monitors/201051) | prod | alert on failed jobs | [#benefits-ee-max-cfi-alerts](https://dsva.slack.com/archives/C06HCLH33PE) |
| [[Synthetics] SANDBOX - Health Check - EP Merge App](https://lighthousedi.ddog-gov.com/monitors/201034) | sandbox | health check | [#benefits-ee-ep-merge-alerts](https://dsva.slack.com/archives/C067L5WK9K6)
| [SANDBOX - EP Merge App - Jobs are Failing](https://lighthousedi.ddog-gov.com/monitors/201050) | sandbox | alert on failed jobs | [#benefits-ee-max-cfi-alerts](https://dsva.slack.com/archives/C06HCLH33PE) |
| [[Synthetics] DEV - Health Check - EP Merge App](https://lighthousedi.ddog-gov.com/monitors/194108) | dev | health check | [#benefits-ee-ep-merge-alerts](https://dsva.slack.com/archives/C067L5WK9K6)

#### Max CFI
[DD Dashboard](https://lighthousedi.ddog-gov.com/dashboard/ama-szd-xn9/employee-experience-max-cfi-vro-platform?fromUser=true&refresh_mode=sliding&tpl_var_namespace%5B0%5D=va-abd-rrd-dev&tpl_var_namespace%5B1%5D=va-abd-rrd-qa&tpl_var_namespace%5B2%5D=va-abd-rrd-sandbox&view=spans&from_ts=1708126548294&to_ts=1708731348294&live=true)

| name | env | purpose | alert channel |
| --- | --- | --- | --- | 
| [[Synthetics] PROD - Health Check - Max CFI](https://lighthousedi.ddog-gov.com/monitors/194869) | prod | health check | [#benefits-ee-max-cfi-alerts](https://dsva.slack.com/archives/C06HCLH33PE) |
| [[Synthetics] SANDBOX - Health Check - Max CFI](https://lighthousedi.ddog-gov.com/monitors/201052) | sandbox | health check | [#benefits-ee-max-cfi-alerts](https://dsva.slack.com/archives/C06HCLH33PE) |
| [[Synthetics] DEV - Health Check - Max CFI](https://lighthousedi.ddog-gov.com/monitors/194880) | dev | health check | [#benefits-ee-max-cfi-alerts](https://dsva.slack.com/archives/C06HCLH33PE) |
